src/frontend/enzymes.jl

```diff
@@ -33,14 +33,17 @@ Base.@kwdef mutable struct Isozyme
 
     "Turnover number for this isozyme catalyzing the forward direction of the
     reaction."
-    kcat_forward::Maybe{Float64} = nothing
+    kcat_forward::Maybe{T} = nothing
 
     "Turnover number for this isozyme catalyzing the reverse direction of the
     reaction."
-    kcat_reverse::Maybe{Float64} = nothing
+    kcat_reverse::Maybe{T} = nothing
 end
 
+const Isozyme = IsozymeT{Float64}
+
 export Isozyme
+export IsozymeT
```

Comment:
add the exports directly under the definitions, this is unnecessary interleaving